src/SWP/Bundle/CoreBundle/Consumer/ContentPushConsumer.php

```diff
@@ -178,15 +178,15 @@ public function doExecute(TenantInterface $tenant, PackageInterface $package): i
 
     protected function findExistingPackage(PackageInterface $package)
     {
-        $existingPackage = null;
         if (null === $package->getEvolvedFrom()) {
-            $existingPackage = $this->packageRepository->findOneBy([
-                'evolvedFrom' => $package->getGuid(),
-            ]);
+            $existingPackage = $this->packageRepository->findOneBy(['guid' => $package->getGuid()]);
+        } else {
```

Comment:
you can use null coalescing operator here `['guid' => $package->getEvolvedFrom() ?? $package->getGuid()]`